packages/api-plugin-fulfillment-method-shipping-flat-rate/src/util/restrictionSchema.js

```diff
@@ -0,0 +1,63 @@
+import SimpleSchema from "simpl-schema";
```

Comment:
Seems like this should get moved to the top-level SimpleSchema file

Reply:
It was copied over from old plugin as such. I shall move this to `/src/simpleSchemas.js`